lib/onebox/engine/whitelisted_generic_onebox.rb

```diff
@@ -86,6 +86,7 @@ def self.default_whitelist
           meetup.com
           mixcloud.com
           mlb.com
+          mp.weixin.qq.com
```

Comment:
This should not be required? We allow oneboxing of all domains unless explicitly blocked.